src/credentials/FabricTable.cpp

```diff
@@ -440,6 +440,18 @@ CHIP_ERROR FabricInfo::VerifyCredentials(const ByteSpan & noc, const ByteSpan &
     return CHIP_NO_ERROR;
 }
 
+CHIP_ERROR FabricInfo::FetchRootPubkey(Crypto::P256PublicKey & outPublicKey) const
+{
+    P256PublicKeySpan publicKeySpan;
+    CHIP_ERROR err = Credentials::ExtractPublicKeyFromChipCert(mRootCert, publicKeySpan);
+    if (err == CHIP_NO_ERROR)
+    {
+        outPublicKey = P256PublicKey(publicKeySpan);
```

Comment:
I wonder whether `P256PublicKey` should have an `operator=` that takes P256PublicKeySpan, to avoid the extra object on the stack here.  Followup for that, please?

Reply:
Done